I suspect the problem in `CI-windows` :
```
Traceback (most recent call last):
  File "C:\Miniconda3\envs\polsys_env\lib\runpy.py", line 185, in _run_module_as_main
    mod_name, mod_spec, code = _get_module_details(mod_name, _Error)
  File "C:\Miniconda3\envs\polsys_env\lib\runpy.py", line 111, in _get_module_details
    __import__(pkg_name)
  File "D:\a\pypolsys\pypolsys\pypolsys\__init__.py", line 22, in <module>
    from .polsys import polsyswrap as polsys
ImportError: DLL load failed while importing polsys: The specified module could not be found.
Error: Process completed with exit code 1.
```
was due to a mix of numpy version between pip and conda. Thus I change the `ci-windows` script to install of all pre-build dependencies with conda (instead of pip through pep517 build-system requirement). It seems to works. I hope this was the origin of the problem...

--- a/.github/workflows/ci-ubuntu.yml
+++ b/.github/workflows/ci-ubuntu.yml
@@ -19,13 +19,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]        
+        python-version: ['3.6', '3.7', '3.8', '3.9', '3.10']        
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
     - uses: actions/checkout@v2
     - name: Set up Python ${{ matrix.python-version }}
-      uses: actions/setup-python@v1
+      uses: actions/setup-python@v2
       with:
         python-version: ${{ matrix.python-version }}
 

--- a/.github/workflows/ci-windows.yml
+++ b/.github/workflows/ci-windows.yml
@@ -27,10 +27,14 @@ jobs:
       - shell: bash -l {0}
         run: |
           conda install m2w64-toolchain libpython
+          # numpy is required for pypolsys building. Better to install it with conda.
+          # Sometime it fails when installed by pip thought `build-system` requirement from `pyproject.toml` 
+          conda install numpy setuptools wheel
       - name: Build
         shell: bash -l {0}
         run: |
-            python -m pip install -e .
+            # Ignore `pyproject.toml` to avoid conda/pip mix in numpy version
+            python -m pip install -e . --no-use-pep517
       - name: Test
         shell: bash -l {0}
         run: |

